samples/subsys/shell/shell_module/prj.conf

```diff
@@ -10,3 +10,7 @@ CONFIG_THREAD_NAME=y
 CONFIG_DEVICE_SHELL=y
 CONFIG_POSIX_CLOCK=y
 CONFIG_DATE_SHELL=y
+CONFIG_USB_DEVICE_STACK=y
```

Comment:
Do we want to enable it by default in this example?

Reply:
i think it should only be enabled in board specific conf file.

Reply:
or configuration overlay